.github/workflows/publish-canary.yml

```diff
@@ -50,8 +52,10 @@ jobs:
           cache_fresh_start: 'true'
 
       # Release
-      - run: |
+      - name: Prerelease Version and Publish
+        run: |
           yarn config set -H 'npmAuthToken' "${{secrets.NPM_TOKEN}}"
           git config --global user.email "engineering@propeldata.com"
           git config --global user.name "Propel Data Cloud, Inc." 
-          COMMIT_MESSAGE=release PRE_RELEASE=true yarn release
+          yarn changeset version --snapshot
```

Comment:
Is it a snapshot or canary? Do they just use `--snapshot` to mean canary interchangeably?

Reply:
The `--snapshot` flag will enable snapshot/canary mode for the command.
They call the dev build a `snapshot`. The term `snapshot` has too many meanings. I personally think that `canary` is a more accurate term for that.